src/gamekit.cs

```diff
@@ -481,8 +602,16 @@ interface GKPlayer : NSSecureCoding {
 		[TV (13,0)][Mac (10,15)][iOS (13,0)]
 		[Export ("scopedIDsArePersistent")]
 		bool ScopedIdsArePersistent { get; }
+
+		[TV (14, 0), NoWatch, Mac (10, 16), iOS (14, 0)]
+		[Field ("GKPlayerIDNoLongerAvailable")]
+		NSString GKPlayerIdNoLongerAvailable { get; }
```

Comment:
we normally remove the prefix on fields, e.g. `IdNoLongerAvailable` when inside the type `GKPlayer`